app/javascript/utils/anchor-docs-headers.ts

```diff
@@ -0,0 +1,65 @@
+import { assetUrl } from './assets'
+import { copyToClipboard } from './copyToClipboard'
+
+export function addAnchorsToDocsHeaders() {
+  if (!window.location.pathname.includes('/docs')) return
```

Comment:
Maybe do this for that little bit of extra checking:
```suggestion
  if (!window.location.pathname.startsWith('/docs')) return
```